editor/plugins/script_editor_plugin.cpp

```diff
@@ -800,7 +802,7 @@ bool ScriptEditor::_test_script_times_on_disk(RES p_for_script) {
 				continue;
 			}
 
-			if (edited_res->get_path() == "" || edited_res->get_path().find("local://") != -1 || edited_res->get_path().find("::") != -1) {
```

Comment:
There's one more occurrence at line 2102.
The check is also used in editor/plugins/shader_editor_plugin.cpp, editor/plugins/script_text_editor.cpp and (partially, so might not be a candidate for changing it) editor/plugins/text_editor.cpp.

But such core changes take more time to review than ad-hoc fixes, so I'd suggest keeping the fix ad-hoc for this bugfix PR, and then make a new PR proposing the addition of `Resource::is_internal`, which might have use cases beyond just checking for built-in scripts too.